.gitlab-ci.yml

```diff
@@ -167,6 +161,21 @@ check-runtime-benchmarks:          &test
     - time cargo check --features runtime-benchmarks
     - sccache -s
 
+build-wasm-release:
+  stage:                          build
+  <<:                             *collect-artifacts
+  <<:                             *docker-env
+  <<:                             *compiler_info
+  # Note: We likely only want to do this for tagged releases, hence the 'only:'
+  only:
+    - /^v[0-9]+\.[0-9]+.*$/        # i.e. v1.0, v2.1rc1
+  script:
+    - time cargo build --locked --target=wasm32-unknown-unknown --manifest-path cli/Cargo.toml --no-default-features --features browser --release
```

Comment:
I'd like to have @expenses's eyes too. Her command was using wasm-pack (see https://github.com/paritytech/devops/issues/401#issuecomment-577231023), and it generates, apart from the .wasm file, some .js/.d.ts glue code.

I'm also not 100% sure if I need to update the .js glue code each time the .wasm file changes or not. If yes, I think they would need to be part of the artifacts too.

Reply:
Yeah, we need the glue code to be generated with the wasm binary, because the function mangling a lot. We should probably be calling `wasm-pack` and not `cargo build` + `wasm-bindgen` here, if possible.

Reply:
Ok cool cheers. I'll move this back into draft and sort that

Reply:
If there are more than 3-4 files, a .zip might be easier.

Reply:
I think better to have the files unzipped and in a directory of their own :) Current code puts them in a directory called 'wasm', which will be available from our s3 bucket on each release. Obviously we'll have to wait for a release to trigger this :) ready for review again now